applications/plugins/CollisionOBBCapsule/examples/benchmark_cubes.pscn

```diff
@@ -0,0 +1,95 @@
+<!-- Requires php install. On 30/11/22, installed with Ubuntu 22.04.01, using php 8.1 https://tecadmin.net/how-to-install-php-on-ubuntu-22-04/  -->
```

Comment:
Is this new scene added on purpose? if yes, maybe just notify it in the PR description